packages/api/lambdas/write-db-dlq-records-to-s3.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const get = require('lodash/get');
+const { s3PutObject, s3ObjectExists } = require('@cumulus/aws-client/S3');
+const { parseSQSMessageBody } = require('@cumulus/aws-client/SQS');
+
+async function handler(event, _) {
```

Comment:
Nit: No need to include `_` if the last argument to the function is unused, I think?